examples/analytics-vendors.amp.html

```diff
@@ -335,6 +336,18 @@
 </amp-analytics>
 <!-- Baidu stats example -->
 
+  <!-- Browsi prediction tracking example -->
+  <amp-analytics type="browsi" id="browsi">
```

Comment:
FYI: The example is throwing an error that " Element "amp-ad[data-amp-slot-index='0']" not found: null​​​". You can fix it by overriding the default config. I'm fine with leaving the error as it is.